src/anisotropic_averaging.cpp

```diff
@@ -226,7 +237,8 @@ void structure_chunk::set_chi1inv(component c, material_function &medium,
   double trivial_val[3] = {0, 0, 0};
   trivial_val[idiag] = 1.0;
   ivec shift1(unit_ivec(gv.dim, component_direction(c)) * (ft == E_stuff ? 1 : -1));
-  LOOP_OVER_VOL(gv, c, i) {
+  PLOOP_OVER_IVECS_C(gv, (gv).little_corner() + (gv).iyee_shift(c),                                   \
```

Comment:
Currently, this loop doesn't seem thread-safe.  It should be changed back to `LOOP_OVER_VOL` until it is fixed.